hack/make/validate-test

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Make sure we're not using gos' Testing package any more in integration-cli
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+IFS=$'\n'
+files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep '^integration-cli/' || true) )
```

Comment:
The `grep` here could be avoided by doing `validate_diff --diff-filter=ACMR --name-only -- 'integration-cli/*.go' || true` instead (ie, let `git` do the hard work).